crates/fuel-core/src/state/rocks_db.rs

```diff
@@ -618,6 +631,29 @@ mod tests {
         )
     }
 
+    #[test]
+    fn open_new_columns() {
+        let tmp_dir = TempDir::new().unwrap();
+
+        // Given
+        let old_columns =
+            vec![Column::Coins, Column::Messages, Column::UploadedBytecodes];
+        let database_with_old_columns =
+            RocksDb::<OnChain>::open(tmp_dir.path(), old_columns.clone(), None)
+                .expect("Failed to open database with old columns");
```

Comment:
Nit: I'm okay with using `unwrap` in tests, if you'd prefer `expect`, it's more standard to explain the reason it should be `Ok`, rather than just telling the user it failed:
https://doc.rust-lang.org/std/result/enum.Result.html#recommended-message-style